pandas/compat/numpy/__init__.py

```diff
@@ -62,9 +62,25 @@ def np_array_datetime64_compat(arr, *args, **kwargs):
     return np.array(arr, *args, **kwargs)
 
 
+def _is_nep18_active():
```

Comment:
We require NumPy>1.16.0, so this can maybe be simplified to

```
IS_NEP18_ACTIVE = not os.environ.get("NUMPY_EXPERIMENTAL_ARRAY_FUNCTION", "1") == "0"
```

See https://numpy.org/neps/nep-0018-array-function-protocol.html#implementation.